src/actions/app.js

```diff
@@ -185,6 +191,16 @@ export function unregisterDragAndDropOverlay(): Action {
 /**
  * Toggle the active tab resources panel
  */
-export function toggleResourcesPanel(): Action {
-  return { type: 'TOGGLE_RESOURCES_PANEL' };
+export function toggleResourcesPanel(): ThunkAction<void> {
+  return (dispatch, getState) => {
+    const isResourcesPanelOpen = getIsActiveTabResourcesPanelOpen(getState());
+    if (isResourcesPanelOpen) {
```

Comment:
So this is a double dispatch, which is bad for performance. It would be better to combine this into a single dispatch that handles both of the actions.

Reply:
Removed the second dispatch, thanks.